Formula/kube-linter.rb

```diff
@@ -0,0 +1,56 @@
+class KubeLinter < Formula
+  desc "Static analysis tool for Kubernetes YAML files and Helm charts"
+  homepage "https://github.com/stackrox/kube-linter"
+  url "https://github.com/stackrox/kube-linter/archive/0.1.2.tar.gz"
+  sha256 "17b0a66536fa6ae4c55eab27f0faaf247002c3eac3d5856fa4c7d2785001be46"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    ENV["GOOS"] = "darwin"
+
+    # Determine XDef variables without relying on the git repository
+    inreplace "./scripts/go-build.sh", "git -C \"${SCRIPT_DIR}/..\" grep -n '//XDef:' -- '*.go'",
+      "grep -nr '//XDef:' -- internal"
+
+    # Remove all functions that rely on the git repository to determine variables
+    inreplace "./scripts/variable_stamps.sh", "gitroot=\"$(git rev-parse --show-toplevel)\"", ""
+    inreplace "./scripts/variable_stamps.sh", "[[ -n \"${gitroot}\" ]] || die \"Could not determine git root\"", ""
+    inreplace "./scripts/variable_stamps.sh", "$(\"${gitroot}/get-tag\")", version.to_s
```

Comment:
Isn't it just easier to use the git repo for the build? Otherwise these options should be upstreamed as well.

Reply:
I went the extra mile because I did not know that you would allow that. Of course that would enable me to remove 3 out of these 4 `inreplace` calls. Do you have any example formula that does this?

Reply:
I didn't know that by using `url` with indented tag and revision I get to work with the actual repo which makes all the pains go away. Today I learnt something 💡